### PR TITLE
Remove VSS Vintorez flash

### LINKED ISSUE #284: Muzzle flash when shooting from VSS Vintorez
When you fire a VSS Vintorez a muzzle flash appears which shouldn't be the case

1. Shoot from VSS Vintorez
2. And you will see the problem

### PR DESCRIPTION
**Pull request by MCT32**
## 🤔 What type of PR is this? (check all applicable)

- [ ] 🍕 Addition
- [ ] ⌨️ Productivity
- [ ] 🐛 Bug Fix
- [ ] 🔥 Optimization
- [ ] ⚙️ Configuration
- [ ] 🌟 Quality Of Life
- [x] ✨ Enhancement
- [ ] 📝 Documentation

## 📝 Description

Removes the muzzle flash from the VSS Vintorez since according to [wikipedia](https://en.wikipedia.org/wiki/AS_Val_and_VSS_Vintorez#Design), the built in suppressor reduces the flash, and in videos of the gun being used online the flash is not visible.

## 🧐 The Rationale

This change simply make the gun behave more realistically.

## 🎯 Key Objectives

Removing the muzzle flash on the VSS Vintorez

## 🚦 Testing 

Shoot the rifle and observe the flash is gone.

## ⏮️ Backwards Compatibility 

Should be

## 📚 Related Issues & Documents

Fixes #284 

## 🖼️ Screenshots/Recordings

https://github.com/Cubed-Development/Modern-Warfare-Cubed/assets/32090502/13e4796b-0fc1-4528-9657-91eb5d74b45e

## 📖 Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 Javadoc
- [ ] 🍕 Comments
- [x] 🙅 no documentation needed

## 😄 [optional] What gif best describes this PR or how it makes you feel?

img

### CODE REVIEW

**Comment by Desoroxxx** [comment]
Missing changelog entry

File: Changelog.md [hunk 1]
```diff
@@ -190,6 +190,7 @@ and this project follows to [Ragnarök Versioning Convention](https://gist.githu
 - Deleted broken Hescobastion and UMP45GFL props
 - Removed the vanilla ability to disable shaders with F4 (this allowed for flash to be disabled)
 - Removed the `WIP` status of the flashbang as it is fully usable and reliable
+- Removed the muzzle flash from the VSS Vintorez to make it more similar to it's real-life counterpart
```
Comment (Desoroxxx) [inline Changelog.md:193]: Should be in the `Fixed` category and be something along the lines of `Fixed VSS Vintorez having a flash when it shouldn't`

**Review by Desoroxxx — CHANGES_REQUESTED** [review]

File: Changelog.md (hunk 1 above)
Comment (Desoroxxx) [reply Changelog.md:193]: Did you forgot to push, this is still an issue

**Comment by Braiam23** [comment]
Thank you very much for the contribution to the mod, but I will move your changes that you made to another Pull request to avoid future conflicts

**Pull request closed without merging**